application/models/team.py

```diff
@@ -11,3 +11,4 @@ class Team(StructuredNode, DefaultPropertyMixin, DefaultHelperMixin):  # pylint:
     name = StringProperty(required=True)
 
     artifacts = RelationshipTo('.Artifact', 'UPLOADED')
+    users = Relationship('.User', 'MEMBER')
```

Comment:
would prefer this to be a directed relationship so Team -[HAS_MEMBER]-> User or User-[IS_MEMBER]-> Team.
That makes the graph more readable and I think it is less error prone when traversing the graph later.